po/da.po

```diff
@@ -14915,51 +14918,68 @@ msgid ""
 "Sometimes includes small exercises where you are asked to expand on the code "
 "in the examples."
 msgstr ""
+"[Rust By Example](https://doc.rust-lang.org/rust-by-example/): dækker "
+"syntaksen i Rust via en række eksempler som viser de forskellige "
+"konstruktioner. Der er sommetider små opgaver hvor du bliver bedt om at "
+"udvide koden i eksemplerne."
 
 #: src/other-resources.md:17
 msgid ""
 "[Rust Standard Library](https://doc.rust-lang.org/std/): full documentation "
 "of the standard library for Rust."
 msgstr ""
+"[Rust Standard Library](https://doc.rust-lang.org/std/): fuld dokumentation "
+"af standardbiblioteket for Rust."
 
 #: src/other-resources.md:19
 msgid ""
 "[The Rust Reference](https://doc.rust-lang.org/reference/): an incomplete "
 "book which describes the Rust grammar and memory model."
 msgstr ""
+"[The Rust Reference](https://doc.rust-lang.org/reference/): en ufuldstændig "
+"bog som beskriver grammatikken og en hukommelsesmodel for Rust."
 
 #: src/other-resources.md:22
 msgid "More specialized guides hosted on the official Rust site:"
-msgstr ""
+msgstr "Mere specialicerede guider hostet på den officielle Rust side:"
 
 #: src/other-resources.md:24
 msgid ""
 "[The Rustonomicon](https://doc.rust-lang.org/nomicon/): covers unsafe Rust, "
 "including working with raw pointers and interfacing with other languages "
 "(FFI)."
 msgstr ""
+"[The Rustonomicon](https://doc.rust-lang.org/nomicon/): dækker unsafe Rust, "
+"inklusiv det at arbejde med rå pointere og at kommunikere med andre sprog "
```

Comment:
pegere? (I don't remember if we already considered whether to use this translation for pointers and it was rejected. In that case, just ignore this.)

Reply:
Not sure either! As far as I recall from my time at university, people didn't use the word "pegere" — but I've heard it a few times later, primarily from people in Copenhagen. So it might be a regional thing :smile: 

I think I would like to keep "pointer" and then we'll see if the Danish Rust community comes after us :smile: